### DISCUSSION TITLE
High precision mouse on Linux feels inconsistent with 8 KHz polling rate

### DISCUSSION THREAD

**Discussion by jansenmtan** [ppy/osu#19351] [Q&A]
### Type

Other

### Bug description

High precision mouse feels inconsistent when using the Razer Viper 8KHz mouse. It feels like the cursor speeds up and slows down randomly. 

Issue not present when using the Razer Viper Ultimate, which has a 1 KHz polling rate.

Issue present on Arch Linux.
`uname -a` : `Linux 5.16.12-zen1-1-zen #1 ZEN SMP PREEMPT Wed, 02 Mar 2022 12:22:53 +0000 x86_64 GNU/Linux`

Issue not present on Windows 10.

### Screenshots or videos

Viper Ultimate @ 1 KHz. 3200 DPI. High precision mouse on.

https://user-images.githubusercontent.com/36340429/180653084-717650fd-74b3-4325-a888-1d4aad1d2685.mp4

Viper 8KHz. 3200 DPI. High precision mouse on.

https://user-images.githubusercontent.com/36340429/180653102-1d3c2e6c-5477-48f6-8697-7fd9d125433c.mp4

Viper 8KHz. 1600 DPI. High precision mouse off.

https://user-images.githubusercontent.com/36340429/180653002-18a9635d-cc38-4df3-82aa-a196e726ae29.mp4

### Version

2022.723.0-lazer

### Logs

[runtime.log](https://github.com/ppy/osu/files/9176012/runtime.log)
[performance.log](https://github.com/ppy/osu/files/9176013/performance.log)

**Comment by peppy** [comment]
Polling at such high rates will likely cause issues at an OS level. It's out of scope for us right now. Please use 1khz polling as anything higher is not actually of benefit.

**Reply by jansenmtan** [reply]
I'll use 1 kHz polling rate devices on Linux. Lazer on Windows seems to work fine with 8 kHz, so I will continue its use in that environment.

> Please use 1khz polling as anything higher is not actually of benefit.

Objectively, there is [about a 3 ms latency improvement](https://youtu.be/px8YWLVb6NA?t=180) that the Viper 8KHz provides over the Viper Ultimate.
For me, the latency improvement is almost imperceptible in-game, but becomes more apparent in high-velocity/high-acceleration cursor movement.
